prosessering-core/src/main/kotlin/no/nav/familie/prosessering/internal/TaskService.kt

```diff
@@ -49,6 +49,11 @@ class TaskService(val taskRepository: TaskRepository) {
     }
 
     fun finnTasksTilFrontend(status: List<Status>, page: Pageable, type: String? = null): List<ITask> {
+        if (status == listOf(Status.FERDIG_NÅ_FEILET_FØR)) {
+            val tasks = if (type == null) taskRepository.findByStatusIn(listOf(Status.FERDIG), page)
+            else taskRepository.findByStatusInAndType(status, type, page)
+            return tasks.filter { it.logg.size > 4 }
```

Comment:
Filteret vil kun virke på gjeldende `page`, er det meningen?

Reply:
Nei, det vil vel vera betre om filteret her går på alle, vi får tru at det ikkje er _så_ mange av oppgåvene som feilar. Da er det kanskje betre å berre sende inn null for page her?

Reply:
enten det, eller lage en ny query som tar hensyn til antall logs for en task.

Reply:
Tror kanskje du trenger å lage en query på dette, men usikker på hvordan man best gjør det. Logg-size er kanskje ikke noe jeg er fan av å sjekke - da det begrenser oss litt for senere.

Pseudokode
```
distinct (task t)
join task_logg tl on t.id = tl.task_id 
where t.status='FERDIG' and tl.type='FEILET' 
```

Er det noe slikt du egentlig ønsker å oppnå her?

Reply:
Sånn det ser ut for meg er fleire enn fire logginnslag godt nok for behovet her, så tenkjer eg at vi heller kan forbetre det seinare viss vi ser behovet.

Reply:
Ikke enig, for det gir litt "magisk" forståelse av task-logg. Kunne godt ha sjekket om loggene inneholder feilet _og_ ferdig i så fall, men du må allikevel hente ut "riktige" tasker i en spørring

Reply:
Det er eit godt poeng, at koden ville bli forvirrande og villeie heller enn rettleie nestemann som kjem inn her og skal lesa og endre noko.

Pusha ny versjon no.